setup.py

```diff
@@ -671,6 +671,7 @@ def run(self):
     "torch/csrc/autograd/aten_variable_hooks.cpp",
     "torch/csrc/autograd/grad_mode.cpp",
     "torch/csrc/autograd/engine.cpp",
+    "torch/csrc/autograd/engine_without_python.cpp",
```

Comment:
We definitely shouldn't have `engine_without_python.cpp`. If you need a no-python version, use `engine.cpp`.